book/src/csi-windows.md

```diff
@@ -0,0 +1,93 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.18 | -
+Beta | 1.19 | -
```

Comment:
It would also be good to add a column for which version of node-driver-registrar and csi-proxy are required

Reply:
done